src/docker/composebuilder/application.go

```diff
@@ -40,15 +40,18 @@ func GetDependenciesDockerConfigs(options ApplicationOptions) (types.DockerConfi
 			if err != nil {
 				return result, err
 			}
-			result[builtDependency.GetContainerName()] = dockerConfig
+			result.Services[builtDependency.GetContainerName()] = dockerConfig
```

Comment:
Unrelated to this PR, but do you think we should start making container name and dependency name synonymous? I think in the past the only reason we made dependency names a concatenation of `#{dependencyName}#{dependencyVersion}` was so that we could support multiple versions of the same dependency. We're not using that now and it's just proving to be a point of confusion